src/components/Search.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component, Fragment } from 'react';
+import { InputAdornment, TextField } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import { Redirect } from 'react-router';
+
+import { SEARCH_PARAM_QUERY } from '../constants';
+
+interface State {
+  query: string;
+  submitted: boolean;
+}
+
+class Search extends Component {
+  state: State = {
+    query: '',
+    submitted: false
+  };
+
+  handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const {
+      target: { value }
+    } = e;
+
+    this.setState({
+      query: value
+    });
+  };
+
+  handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    this.setState({
+      submitted: true
+    });
+  };
+
+  render() {
+    const { submitted, query } = this.state;
+    return (
+      <Fragment>
```

Comment:
May be able to use shorthand here `<>`

Reply:
[updated](https://github.com/CodeForBoulder/upswyng/pull/75/commits/edfdd51ccba1b9ba9ef73c893c3eb4e8d906be1e) 👍